include/alp/reference/matrix.hpp

```diff
@@ -863,6 +863,18 @@ namespace alp {
 				typedef storage::polynomials::FullFactory<> factory_type;
 		};
 
+		/** Specialization for hermitian tridiagonal matrix */
+		template< enum Backend backend >
+		struct determine_poly_factory< structures::HermitianTridiagonal, imf::Id, imf::Id, backend > {
+
+			private:
+				using interval = std::tuple_element< 0, structures::SymmetricTridiagonal::band_intervals >::type;
```

Comment:
What is this used for?

Reply:
It will be used in the commented out line 847 once band storage is added.